cornellh4i/ctw-admin
# Styled data viz page + refactored collapsible

## Overview

Closes #35 : Styled the data viz page based on the hifis and refactored the collapsible components to remove redundancies

## Changes Made

Styling the data viz page to match the hifis involved adding/updating css files to components and refactoring the dropdown components. I refactored a separate Collapsible component that can be used for all of the dropdowns and removed redundancies in the previous code (i.e. removed LocationSelector and updated css). I also added a SilderSelector component for Time Frame and Altitude. Note that as of now we do not have a Time Frame component, so the Altitude component is being used as a placeholder in this dropdown.

## Test Coverage

1) checkout style-data-viz
2) go to http://localhost:3000/dataviz
4) observe dropdowns: Mesh Type, Location, Time Frame, Altitude (Note: currently placeholder data)
